src/modules/commander/Commander.cpp

```diff
@@ -2081,6 +2081,9 @@ Commander::run()
 					   ) {
 						print_reject_arm("Not arming: Switch to a manual mode first");
 
+					} else if (vtol_status.vtol_in_trans_mode) {
```

Comment:
You should be able to delete this part now.

Reply:
Yep, sorry, forgot to remove that.